### PR TITLE
Add `evil-collection-ebuku`.

### PR DESCRIPTION
**Pull request by flexibeast**
Further to [this user request](https://github.com/flexibeast/ebuku/issues/8), i'd like to add support for Ebuku to evil-collection.

i only use evil-local-mode, myself, and only occasionally at that, so i'm not sure of the extent to which the Evil keybindings should differ from the non-Evil ones, given the evil-collection guidelines. Any recommendations in this regard would be much appreciated. :-)

### CODE REVIEW

File: modes/ebuku/evil-collection-ebuku.el [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+;; Evil bindings for Ebuku.
+
+;;; Code:
+
+(require 'ebuku)
+(require 'evil-collection)
+
+
+;;;###autoload
+(defun evil-collection-ebuku-setup ()
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: Ebuku generally uses single keys right? I'm not too familiar with the package so keep that in mind. :)

File: modes/ebuku/evil-collection-ebuku.el [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "Set up `evil' bindings for `Ebuku'."
+  (evil-collection-define-key 'normal 'ebuku-mode-map
+    "q" 'quit-window
+    "a" 'ebuku-add-bookmark
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: This is probably ok since there's no editing.

File: modes/ebuku/evil-collection-ebuku.el [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "d" 'ebuku-delete-bookmark
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: Maybe "x" as well. Up to you.

File: modes/ebuku/evil-collection-ebuku.el [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    "e" 'ebuku-edit-bookmark
+    "g" 'ebuku-refresh
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: "gr"

File: modes/ebuku/evil-collection-ebuku.el [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    "n" 'ebuku-next-bookmark
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: Maybe "j" or "C-j"?

I can see an evil user wanting to use "n" for when they're searching for text.

File: modes/ebuku/evil-collection-ebuku.el [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    "p" 'ebuku-previous-bookmark
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: Inverse to j and c-j, "k" and "C-k", maybe.

File: modes/ebuku/evil-collection-ebuku.el [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    "r" 'ebuku-search-on-recent
+    "s" 'ebuku-search
+    "*" 'ebuku-show-all
+    "-" 'ebuku-toggle-results-limit
```
Comment (jojojames) [inline modes/ebuku/evil-collection-ebuku.el]: Maybe "t" for toggle, "-" is probably ok too.

File: modes/ebuku/evil-collection-ebuku.el (hunk 1 above)
Comment (flexibeast) [reply modes/ebuku/evil-collection-ebuku.el]: Yes, only single keys at this point, and i have no immediate plans to change that.

File: modes/ebuku/evil-collection-ebuku.el (hunk 3 above)
Comment (flexibeast) [reply modes/ebuku/evil-collection-ebuku.el]: Thanks, added.

File: modes/ebuku/evil-collection-ebuku.el (hunk 4 above)
Comment (flexibeast) [reply modes/ebuku/evil-collection-ebuku.el]: Done.

File: modes/ebuku/evil-collection-ebuku.el (hunk 5 above)
Comment (flexibeast) [reply modes/ebuku/evil-collection-ebuku.el]: Changed "n" to "j", and added "C-j" as well.

File: modes/ebuku/evil-collection-ebuku.el (hunk 6 above)
Comment (flexibeast) [reply modes/ebuku/evil-collection-ebuku.el]: Changed "p" to "k", and added "C-k" as well.

File: modes/ebuku/evil-collection-ebuku.el (hunk 7 above)
Comment (flexibeast) [reply modes/ebuku/evil-collection-ebuku.el]: i'd like to keep "-", but i've added "t" as well.

**Comment by jojojames** [comment]
Thanks!

**Comment by flexibeast** [comment]
Thanks for the helpful feedback!

**Comment by jojojames** [comment]
No problem, thanks for the contribution and the quick turnaround. :)

**Pull request merged**